mgmt/rest/v2/config.go

```diff
@@ -159,3 +205,66 @@ func getPluginType(t string) (core.PluginType, error) {
 	}
 	return ityp, nil
 }
+
+// deletePluginConfigItemHelper builds different forms of request data into the way method deletePluginConfigItem accepts.
+// currently it accepts go-swagger client, swagger-ui and SanpCLI.
+func deletePluginConfigItemHelper(r *http.Request) []string {
+	buf, _ := ioutil.ReadAll(r.Body)
+	rdr2 := ioutil.NopCloser(bytes.NewBuffer(buf))
+
+	dm := map[string][]string{}
+	err := json.Unmarshal(buf, &dm)
+
+	sw := false
+	if err != nil {
+		// go-swagger sends url-encoded form data.
+		// Unescaping is necessary.
+		data, _ := url.QueryUnescape(string(buf))
+		tokens := strings.Split(data, "=")
+		if len(tokens) == 2 {
+			itokens := strings.Split(tokens[1], ",")
+			dm["config"] = itokens
+			sw = true
+		}
+	} else {
+		sw = true
+	}
+
+	var src []string
+	if sw {
+		src = dm["config"]
+	} else {
+		r.Body = rdr2
+	}
+	return src
+}
+
+// setPluginConfigItemHelper builds different forms of request data into the way method setPluginConfigItem accepts.
+// currently it accepts go-swagger client, swagger-ui and SanpCLI.
+func setPluginConfigItemHelper(r *http.Request) {
+	buf, _ := ioutil.ReadAll(r.Body)
+	rdr2 := ioutil.NopCloser(bytes.NewBuffer(buf))
```

Comment:
do we need intermediate variable here? It couldn't be assigned to r.Body in line 268?